unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Transform/ECSTransformPlugin.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using DCL.ECSRuntime;
+
+namespace DCL.ECSComponents
+{
+    public class ECSTransformPlugin : IDisposable
+    {
+        private readonly ECSComponentsFactory factory;
+        private readonly int componentId;
+
+        public ECSTransformPlugin(int componentId, ECSComponentsFactory factory)
+        {
+            ECSTransformHandler handler = new ECSTransformHandler();
+            factory.AddOrReplaceComponent(componentId, ECSTransformSerialization.Deserialize, () => handler);
+            this.factory = factory;
+            this.componentId = componentId;
+        }
```

Comment:
Not sure if it's a good idea, but this starts to look like we could implement a `ECSComponentPlugin<THandler>` class.

Reply:
yeah, that might not be a bad idea, we would have to analyze it to make sure that won't constrain new components, since not every component may need to be created in the same standardized way